doc/nrf-bm/migration/nrf5_bm_migration.rst

```diff
@@ -91,6 +87,22 @@ In |BMshort|, there are a few ready-made partitioning schemes that can be select
 Partitioning can be tweaked by making simple changes to textual **Devicetree** files which define the layout of the memory.
 These can be edited in the board files, or applied to existing boards as **overlays**.
 
+Non-volatile Memory (NVM)
+*************************
+
+The nRF51 and nRF52 devices supported by the nRF5 SDK utilized flash memory as their non-volatile memory (NVM) technology for storing program code and data.
+In contrast, the nRF54L devices employ RRAM, a different NVM technology.
+The inherent differences between flash and RRAM technologies impact the way storage and file systems are implemented.
```

Comment:
Perhaps we could clarify what kind of storage we are talking about:
`...the way non-volatile storage libraries, including filesystems, are implemented`